ketcher-autotests/tests/User-Interface/Editing-Tools/Copy-Cut-Paste/copy-cut-paste.spec.ts

```diff
@@ -298,8 +305,15 @@ test.describe('Copy/Cut/Paste Actions', () => {
     await waitForRender(page, async () => {
       await clickOnBond(page, BondType.SINGLE, 0);
     });
-    await page.keyboard.press('Control+c');
-    await page.keyboard.press('Control+v');
+    await waitForSpinnerFinishedWork(
+      page,
+      async () => await page.keyboard.press('Control+c'),
+    );
+
+    await waitForSpinnerFinishedWork(page, async () =>
+      page.keyboard.press('Control+v'),
+    );
```

Comment:
I would propose to wrap copy/cut/paste into helpers. to not write everytime 
```
 await waitForSpinnerFinishedWork(page, async () =>
      page.keyboard.press('Control+v'),
    );
```

But write just

```
await copy(page)
await paste(page)
```